### ISSUE TITLE
disable current file context does not have an appropriate aria label

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#236076]
discovered here https://github.com/microsoft/vscode/issues/235203#issuecomment-2541804643 cc @rperez030  

1. with a screen reader enabled, tab to the attached file 
2. tab again to reach the button to toggle the state
3. 🐛 it does not indicate the action that will be taken if clicked

<img width="957" alt="Image" src="https://github.com/user-attachments/assets/be8d7302-27bf-4da6-bd79-44c68bb97c5d" />

<img width="950" alt="Image" src="https://github.com/user-attachments/assets/f92e704e-a9f6-4371-aca0-09d30bfdb13d" />
